### PR TITLE
JENKINS-44888 Link commit lozenge to Changes tab on pipeline run

### PR DESCRIPTION
**Pull request by sophistifunk**
Make the commit lozenge on activities tab link directly to changes tab in run details popup.

# Description

See [JENKINS-44888](https://issues.jenkins-ci.org/browse/JENKINS-44888).

# Submitter checklist
- [X] Link to JIRA ticket in description, if appropriate.
- [X] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by sophistifunk** [comment]
changes-url

File: blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx [hunk 1]
```diff
@@ -132,7 +133,7 @@ ActivityDetailsRow.propTypes = {
     t: PropTypes.func,
     getTimes: PropTypes.func,
     columns: PropTypes.object,
-    isMultibranch: PropTypes.boolan,
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx]: dear god

**Comment by michaelneale** [comment]
if its green get it in. that horrifying if-we-could-type-check typo alone is worth it...

**Comment by michaelneale** [comment]
need to look closer - but there are a lot of test failures and the screen shot show the app not loading

https://ci.blueocean.io/job/blueocean/job/feature%252FJENKINS-44888-commit-lozenge-link/2/artifact/acceptance-tests/target/screenshots/io.blueocean.ath.live.creation.GitCreationTest_testHttpsPrivateRepository.png

looks like something isn't quite right (never seen it fail like this without it being legit)

**Comment by sophistifunk** [comment]
Weird. Also, Future Funk's problem. I have tax stuff to sort.

**Comment by cliffmeyers** [comment]
This is the same kind of catastrophic failure as I've been seeing in #1406

**Comment by michaelneale** [comment]
if you update to master again - should be better/healthier

**Comment by michaelneale** [comment]
actually maybe not - nothing is loading in ATH here...

**Comment by michaelneale** [comment]
ok this is, and I use the technical term "shit". 

It fails only on CI, that specific selenium. I think this is the same as what @cliffmeyers has seen. We have to ignore this one for now, but it is a good test case

**Comment by sophistifunk** [comment]
Fixed the typo that we suspect is breaking the build, let's see how it goes.

**Comment by sophistifunk** [comment]
Holy crap it's green.

**Pull request merged**